src/ast/astnodes.ts

```diff
@@ -23,98 +23,208 @@ AST.prototype.tp$name = "AST";
 /* ---------------------- */
 
 /* ----- expr_context ----- */
-export class expr_context extends AST {}
+export class expr_context extends AST {
+    kind = 0;
```

Comment:
why this odd mixture of `_kind` and `kind`?

Reply:
Just a mistake. I had all kind initially and then typescript complained because a Constant already has a kind property. And then I must have missed a few on changing it to _kind